packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js

```diff
@@ -0,0 +1,13 @@
+// @flow strict
+import React from 'react';
+import { Box, Popover } from 'gestalt';
```

Comment:
I don't think we want to change whatever renaming people have been doing. I think this codemod should only update the import, but not alter the renaming aspect

Reply:
I agree. That's how all renaming codemods we've been doing work though. Row/Stack, ScrollableContainer, GestaltProvider, etc.  I tried trying to keep the alias but I couldn't figure out. I'd try again. If it's too cumbersome, I think we just need to grep for `Flyout as` and see the use cases.

Reply:
DONE, I kept aliases.